Wbooks/.env

```diff
@@ -0,0 +1 @@
+API_BASE_URL=http://private-anon-72e517d985-woloxrailstraining.apiary-mock.com/api/v1
```

Comment:
Remove this file from repo and add it to .gitignore

Reply:
I did it like this originally , I don't know why it was uploaded